torch_xla/core/dynamo_bridge.py

```diff
@@ -390,6 +390,18 @@ def call_module(self, target, args, kwargs):
 
 
 def extract_compiled_graph(xla_model, xla_args):
```

Comment:
one small optimization: after we trace for fallback, if none, we just can execute (todo, in later pr)